agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -361,3 +360,70 @@ class TraceDB(Document):
 
     class Settings:
         name = "traces"
+
+
+class OldEvaluationTypeSettings(BaseModel):
+    similarity_threshold: Optional[float]
+    regex_pattern: Optional[str]
+    regex_should_match: Optional[bool]
+    webhook_url: Optional[str]
+    llm_app_prompt_template: Optional[str]
+    custom_code_evaluation_id: Optional[str]
+    evaluation_prompt_template: Optional[str]
+
+
+class OldEvaluationScenarioInput(BaseModel):
+    input_name: str
+    input_value: str
+
+
+class OldEvaluationScenarioOutput(BaseModel):
+    variant_id: str
+    variant_output: str
+
+
+class OldEvaluationDB(Document):
+    app: Link[AppDB]
+    organization: Link[OrganizationDB]
+    user: Link[UserDB]
+    status: str
+    evaluation_type: str
+    evaluation_type_settings: OldEvaluationTypeSettings
+    variants: List[PydanticObjectId]
+    testset: Link[TestSetDB]
+    created_at: Optional[datetime] = Field(default=datetime.utcnow())
+    updated_at: Optional[datetime] = Field(default=datetime.utcnow())
+
+    class Settings:
+        name = "evaluations"
+
+
+class OldEvaluationScenarioDB(Document):
+    user: Link[UserDB]
+    organization: Link[OrganizationDB]
+    evaluation: Link[EvaluationDB]
+    inputs: List[OldEvaluationScenarioInput]
+    outputs: List[OldEvaluationScenarioOutput]  # EvaluationScenarioOutput
+    vote: Optional[str]
+    score: Optional[Any]
+    correct_answer: Optional[str]
+    created_at: Optional[datetime] = Field(default=datetime.utcnow())
+    updated_at: Optional[datetime] = Field(default=datetime.utcnow())
+    is_pinned: Optional[bool]
+    note: Optional[str]
+
+    class Settings:
+        name = "evaluation_scenarios"
+
+
+class OldCustomEvaluationDB(Document):
```

Comment:
let's copy everything to that file as discussed

Reply:
Done.